docs/3.main-concepts/9_1.webchat.mdx

```diff
@@ -0,0 +1,90 @@
+---
+name: 10. Webchat
+route: /main-concepts/webchat
+menu: 3. Main Concepts
+---
+
+# **Webchat**
+
+## What is Webchat?
+
+Bots developed with Botonic can be easily deployed to different messaging channels. 
+This is due to the fact that Botonic components are specifically treated in order to be executed in a server environment such as Node.Js. 
+As they are based on React, Botonic components are the perfect fit in order to offer you applications such as Webchat. Webchat is a system that allows you to not being restricted to use one of the different messaging providers in order to be able to run your bots and it can easily be embedded in your website.
```

Comment:
>Webchat is a system that allows you to not being restricted to use one of the different messaging providers in order to be able to run your bots and it can easily be embedded in your website.

Webchat is a system that allows you to embed your bots in a web page without requiring any third party messaging provider